luminoso_api/client.py

```diff
@@ -487,6 +487,16 @@ def get_raw(self, path, **params):
         url = ensure_trailing_slash(self.url + path.lstrip('/'))
         return self._request('get', url, params=params).text
 
+    def save_to_file(self, path, filename, **params):
+        """
+        Saves binary content to file filename. Useful for downloading .xlsx
```

Comment:
"to file filename" is kind of... weird.  I'm not sure what it should be, though.